public/locales/fr/gamepage.json

```diff
@@ -1,5 +1,10 @@
 {
   "box": {
+    "change": {
```

Comment:
I guess we will need to ask for those missing translations to the guys on the PR.